pyramid_hypernova/batch.py

```diff
@@ -53,10 +56,11 @@ def create_job_groups(jobs, max_batch_size):
 
 class BatchRequest(object):
 
-    def __init__(self, batch_url, plugin_controller, max_batch_size=None):
+    def __init__(self, batch_url, plugin_controller, json_encoder, max_batch_size=None):
```

Comment:
Could we give `json_encoder` a default value, or make it optional? This is likely to break any consumer that implements a custom `pyramid_hypernova.batch_request_factory`.

Reply:
Made it optional with the generic `JSONEncoder` as the default value.